app/api/__init__.py

```diff
@@ -1,6 +1,8 @@
 from flask import Blueprint
+from flask_sqlalchemy import SQLAlchemy
 
 bp = Blueprint('api', __name__)
+db = SQLAlchemy()
```

Comment:
Are these lines necessary if you're importing `db` from `app`?

Reply:
I suppose not. The only reason I included that line was because I assumed it was needed to define `db`, which would then be referenced by `routes.py`.